TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs

```diff
@@ -6,16 +6,29 @@
 using Microsoft.AspNetCore.Mvc;
 using Microsoft.AspNetCore.Mvc.RazorPages;
 using TheModernIlluminati.Models;
+using System.Globalization;
 
 namespace TheModernIlluminati.Pages
 {
     public class SearchCountryModel : PageModel
     {
+        public string _CountrySearch;
+
+        TextInfo myTI = new CultureInfo("en-US", false).TextInfo;
         public bool searchFinished { get; set; }
         [BindProperty]
-        public string CountrySearch { get; set; }
+
+        
+        public string CountrySearch { get; set;}
         public string CountryCode { get; set; }
-        public void OnGet()
+
+       
+    public static string ToTitleCase(string title)
```

Comment:
Api call is case sensitive with country name, hence refactored the user input according to the api call, which is only returning expected response with case capitalizing the first character.
https://martinfowler.com/articles/richardsonMaturityModel.html